woodwork/tests/conftest.py

```diff
@@ -186,11 +186,6 @@ def numeric_time_index_df_koalas(numeric_time_index_df_pandas):
     return ks.from_pandas(numeric_time_index_df_pandas)
 
 
-@pytest.fixture(params=['numeric_time_index_df_pandas', 'numeric_time_index_df_dask', 'numeric_time_index_df_koalas'])
-def numeric_time_index_df(request):
```

Comment:
I think we might actually want to keep this fixture. The DataTables test that was using it doesn't have an equivalent in the Accessor, and I think we'd benefit from having the equivalent test. I can put up a quick PR so that the tests pass when keeping this fixture

Reply:
Added this test to Accessor in #716

Reply:
merged!

Reply:
Okay, updated branch